packages/desktop-client/src/components/reports/graphs/BarGraph.tsx

```diff
@@ -32,7 +34,7 @@ type PayloadChild = {
 };
 
 type PayloadItem = {
-  value: string;
+  value?;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏what's going on here? Looks like it's missing a type.

Reply:
It's depreciated - from a previous version of the data structure. I've just removed it because it is not used.